### PR TITLE
blockchain/vm: modify opcode computation costs for istanbulCompatible change

### PR DESCRIPTION
**Pull request by yoomee1313**
## Proposed changes
Some opcode computation costs are modified and it is activated with istanbulCompatible change activation.
- Modification1. Reset computation cost modification of `sload`, `balance`, `extcodehash`. It is the revert of the computation cost modification which is applied at #958.
- Modification2. Redefine computation cost of `selfBalance` and `chainId`  based on the execution time.
- Modification3. Redefine computation cost of `addmod`, `mulmod`, `not`, `xor`, `shl`, `shr`, `sar` based on the execution time.

The computation cost is calculated based on the instructions_test.go benchmark test. Double the average execution time when benchtime is 200x.
```
ubuntu@ip-10-63-239-152:~/klaytn/blockchain/vm$ go test -bench=SelfBalance -benchtime=200x
goos: linux
goarch: amd64
pkg: github.com/klaytn/klaytn/blockchain/vm
cpu: Intel(R) Xeon(R) Platinum 8275CL CPU @ 3.00GHz
BenchmarkOpSelfBalance-8   	     200	       186.2 ns/op
PASS
ok  	github.com/klaytn/klaytn/blockchain/vm	0.189s
```

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by aidan-kwon — APPROVED** [review]

File: blockchain/vm/jump_table.go [hunk 1]
```diff
@@ -106,7 +107,7 @@ func newConstantinopleInstructionSet() JumpTable {
 		constantGas:     params.ExtcodeHashGasConstantinople,
 		minStack:        minStack(1, 1),
 		maxStack:        maxStack(1, 1),
-		computationCost: params.ExtCodeHashComputationCostConstantinople,
+		computationCost: params.ExtCodeHashComputationCost,
```
Comment (jeongkyun-oh) [inline blockchain/vm/jump_table.go:110]: Doesn't have any problem before `IstanbulCompatibleBlock`?

Comment (yoomee1313) [reply blockchain/vm/jump_table.go:110]: It is the revert of the computation cost modification which is applied at https://github.com/klaytn/klaytn/pull/958 :)

Comment (hqjang-pepper) [reply blockchain/vm/jump_table.go:110]: I think yumiel set both ExtCodeHashComputationCostConstantinople and ExtCodeHashComputationCostEIP1884 to 1000 for the simplexity

File: params/computation_cost_params.go [hunk 2]
```diff
@@ -78,6 +73,7 @@ const (
 	MloadComputationCost          = 376
 	MstoreComputationCost         = 288
 	Mstore8ComputationCost        = 5142
+	SloadComputationCost          = 835
```
Comment (hqjang-pepper) [inline params/computation_cost_params.go:76]: Some opcodes were set to apply the value of **EIP150** among EIP150 and EIP 1884, is there any reason for that?

Comment (yoomee1313) [reply params/computation_cost_params.go:76]: It is the revert of the computation cost modification which is applied at #958 :)

**Review by hqjang-pepper — APPROVED** [review]

**Review by jeongkyun-oh — APPROVED** [review]

**Comment by jeongkyun-oh** [comment]
@yoomee1313 It would be better to reference the PR which you want to revert.

**Review by winnie-byun — APPROVED** [review]

**Pull request merged**